.travis.yml

```diff
@@ -15,4 +15,4 @@ matrix:
     - env: VERSION='-PhadoopVersion=2.9.0'
     - env: VERSION='-PhadoopVersion=3.0.0'
 script:
-  - ./gradlew clean build licenseMain licenseTest findbugsMain findbugsTest buildRpm $VERSION
```

Comment:
Oh man. Sorry, @pauljv92, the $VERSION is definitely still needed as well.

Reply:
gotchu, will fix it